cvmfs/catalog_mgr_impl.h

```diff
@@ -1,3 +1,13 @@
+/*
+ * catalog_mgr_impl.h
+ *
+ *  Created on: Aug 13, 2015
+ *      Author: molina
+ */
+
+#ifndef CVMFS_CATALOG_MGR_IMPL_H_
+#define CVMFS_CATALOG_MGR_IMPL_H_
+
 /**
```

Comment:
Very true. This is fixed in the pull request number 3, where I have the cpplint fixes